README.md

```diff
@@ -9,7 +9,7 @@ Navigation: [Website][1] | **Server repository** | [Client repository][2]
 [![Build Status](https://travis-ci.org/Zarel/Pokemon-Showdown.svg)](https://travis-ci.org/Zarel/Pokemon-Showdown)
 [![Dependency Status](https://david-dm.org/zarel/Pokemon-Showdown.svg)](https://david-dm.org/zarel/Pokemon-Showdown)
 [![devDependency Status](https://david-dm.org/zarel/Pokemon-Showdown/dev-status.svg)](https://david-dm.org/zarel/Pokemon-Showdown#info=devDependencies)
-
+[![Deploy to Heroku](https://www.herokucdn.com/deploy/button.png)](https://heroku.com/deploy)
```

Comment:
This doesn't seem to be enough to deploy PS specifically, unless it actually reads the referer? Even so I think HTTPS links cut off the referer at the domain.